packages/imperative/src/utilities/src/CliUtils.ts

```diff
@@ -484,26 +484,15 @@ export class CliUtils {
             secToWait = maxSecToWait;
         }
 
-        return new Promise((resolve, reject) => {
-            require("read")({
-                input: process.stdin,
-                output: process.stdout,
-                terminal: true,
-                prompt: message,
-                silent: opts?.hideText,
-                replace: opts?.maskChar,
-                timeout: secToWait ? (secToWait * 1000) : null  // eslint-disable-line @typescript-eslint/no-magic-numbers
-            }, (error: any, result: string) => {
-                if (error == null) {
-                    resolve(result);
-                } else if (error.message === "canceled") {
-                    process.exit(2);
```

Comment:
Do we need to await the promise and wrap it in a try-catch block? The `process.exit` may be important in order for Ctrl+C to be handled properly when using daemon mode.